### PR TITLE
 blockdev_commit_with_stop:commit with on-error:stop

### PR DESCRIPTION
**Pull request by aliang123**
Commit with on-error:stop

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2120241

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.blockdev_commit_with_stop.q35: PASS (61.15 s)

File: qemu/tests/blockdev_commit_with_stop.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        self.files_info.append([root_dir, filename])
+
+    def commit_snapshots(self):
+        device = self.params["device_tag"].split()[0]
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        device = self.get_node_name(snapshot_tags[-1])
+        arguments = {}
+        arguments.update({"on-error": "stop"})
```
Comment (hereischen) [inline qemu/tests/blockdev_commit_with_stop.py:23]: what about?
`arguments = {"on-error": "stop"}`

Comment (hereischen) [reply qemu/tests/blockdev_commit_with_stop.py:23]: or 
`cmd, arguments = backup_utils.block_commit_qmp_cmd(device, on-error="stop")`

**Review by hereischen — APPROVED** [review]

File: qemu/tests/blockdev_commit_with_stop.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_commit_with_stop.py]: thanks @hereischen, have updated.

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng, please help to review, thanks.

**Comment by zhencliu** [comment]
also add @nickzhq

**Comment by nickzhq** [comment]
LGTM.

**Comment by XueqiangWei** [comment]
@zhencliu @MiriamDeng, please help to review, thanks.

**Comment by MiriamDeng** [comment]
LGTM
 (2/2) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_commit_with_stop: PASS (141.98 s)

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**